moveit_ros/benchmarks/src/simple_benchmarks/CombinePredefinedPosesBenchmark.cpp

```diff
@@ -1,156 +0,0 @@
-/*********************************************************************
```

Comment:
Do we have a replacement for this benchmark?

Reply:
I'd like to move this into the tutorial repository. What do you think about that?